src/rpc/blockchain.cpp

```diff
@@ -1389,7 +1391,7 @@ UniValue MempoolInfoToJSON(const CTxMemPool& pool)
     ret.pushKV("maxmempool", (int64_t) maxmempool);
     ret.pushKV("mempoolminfee", ValueFromAmount(std::max(pool.GetMinFee(maxmempool), ::minRelayTxFee).GetFeePerK()));
     ret.pushKV("minrelaytxfee", ValueFromAmount(::minRelayTxFee.GetFeePerK()));
-
+    ret.pushKV("unbroadcastsize", static_cast<int64_t>(pool.GetUnbroadcastTxs().size()));
```

Comment:
```suggestion
    ret.pushKV("unbroadcastsize", int64_t{pool.GetUnbroadcastTxs().size())};
```

This does the same thing, but has additional compile time checks that fail compilation when the conversion comes with a loss of precision.

Reply:
Really sorry about this - I got a compilation error: 
```
rpc/blockchain.cpp:1394:43: error: non-constant-expression cannot be narrowed from type 'std::__1::set<uint256, std::__1::less<uint256>,
      std::__1::allocator<uint256> >::size_type' (aka 'unsigned long') to 'int64_t' (aka 'long long') in initializer list [-Wc++11-narrowing]
    ret.pushKV("unbroadcastsize", int64_t{pool.GetUnbroadcastTxs().size()});
                                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
rpc/blockchain.cpp:1394:43: note: insert an explicit cast to silence this issue
    ret.pushKV("unbroadcastsize", int64_t{pool.GetUnbroadcastTxs().size()});
                                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                          static_cast<int64_t>(          )
```
So I followed the suggestion to fix it. I'm not sure why it didn't work for me.

Reply:
Oh, I guess you can use this signature `bool pushKV(const std::string&, uint64_t)`. It should already be selected by default, so I don't see why we want to cast in the first place.

I think you can either:

* Remove the cast
* Select the signature for `uint64_t`